src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -136,4 +163,20 @@ const useClickTrackerHandler = (props = {}) => {
   );
 };
 
+export const useConstructLiteSiteATIEventTrackUrl = (props = {}) => {
+  const atiTrackingParams = useExtractTrackingProps(props);
+  const atiClickTrackingUrl = buildATIEventTrackUrl(atiTrackingParams);
+  return atiClickTrackingUrl;
+};
+
+export const useATIClickTrackerHandler = (props = {}) => {
+  const { isLite } = useContext(RequestContext);
+  const canonicalHandler = useClickTrackerHandler(props);
```

Comment:
Would it be best to just call it a `clickTrackerHandler` instead of including canonical in the name?

```suggestion
  const clickTrackerHandler = useClickTrackerHandler(props);
```

Reply:
Yep good point. I guess it would be better to think of this as 'everything but lite'. Hopefully this is clear through the logic, so the variable name doesn't need to specify canonical.